package.json

```diff
@@ -47,32 +47,35 @@
   "private": true,
   "resolutions": {
     "minimist": "^1.2.5",
-    "webdriver-manager": "^12.1.8"
+    "webdriver-manager": "^12.1.8",
+    "ts-node": "10.2.1"
   },
   "dependencies": {
-    "@angular/animations": "~11.2.14",
-    "@angular/cdk": "^11.2.13",
-    "@angular/common": "~11.2.14",
-    "@angular/compiler": "~11.2.14",
-    "@angular/core": "~11.2.14",
-    "@angular/forms": "~11.2.14",
-    "@angular/localize": "11.2.14",
-    "@angular/platform-browser": "~11.2.14",
-    "@angular/platform-browser-dynamic": "~11.2.14",
-    "@angular/platform-server": "~11.2.14",
-    "@angular/router": "~11.2.14",
-    "@kolkov/ngx-gallery": "^1.2.3",
-    "@ng-bootstrap/ng-bootstrap": "9.1.3",
-    "@ng-dynamic-forms/core": "^13.0.0",
-    "@ng-dynamic-forms/ui-ng-bootstrap": "^13.0.0",
-    "@ngrx/effects": "^11.1.1",
-    "@ngrx/router-store": "^11.1.1",
-    "@ngrx/store": "^11.1.1",
-    "@nguniversal/express-engine": "11.2.1",
+    "@angular/animations": "~13.2.6",
+    "@angular/cdk": "^13.2.6",
+    "@angular/common": "~13.2.6",
+    "@angular/compiler": "~13.2.6",
+    "@angular/core": "~13.2.6",
+    "@angular/forms": "~13.2.6",
+    "@angular/localize": "13.2.6",
+    "@angular/platform-browser": "~13.2.6",
+    "@angular/platform-browser-dynamic": "~13.2.6",
+    "@angular/platform-server": "~13.2.6",
+    "@angular/router": "~13.2.6",
+    "@babel/runtime": "^7.17.2",
+    "@kolkov/ngx-gallery": "^2.0.1",
+    "@material-ui/core": "^4.11.0",
+    "@material-ui/icons": "^4.9.1",
+    "@ng-bootstrap/ng-bootstrap": "^11.0.0",
+    "@ng-dynamic-forms/core": "^14.0.1",
```

Comment:
@ybnd 
it seems that has been released version 15.0.0 with support to angular 13, could you update it ?

Reply:
v15 also requires `rxjs@7` & there's some issues with the upgrade I haven't figured out yet

[See comment ↓](https://github.com/DSpace/dspace-angular/pull/1567#issuecomment-1093048741)

Reply:
@ybnd : In that case, I'd recommend we treat that task as a separate ticket.  We should move this PR forward "as-is" (with regards to ng-dynamic-forms & rxjs), and create a ticket to address those in a separate PR.  If you'd be willing to create a separate ticket for that effort, I'd appreciate it.  If you are interested in working on that, I can also assign it to you (though I'd appreciate a rough estimate of how many hours work it's likely to be)... but it should be in a follow-up PR.

Reply:
@tdonohue if we move this to a separate PR it would make sense for me to keep working on it since I've already started.

@artlowel suggested an 8h time estimate.

Reply:
@ybnd : I've created a separate ticket for this work & assigned to you: #1590  So, you can continue that work, just create a follow-up PR for it.   (The reason I want this in a separate ticket/PR is that it's not exactly required for the Angular 13 upgrade to "work", and I want to ensure this Angular 13 PR can be merged ASAP as it's very large and may impact many other Angular PRs.)

@atarix83 : Please re-review this PR, keeping in mind that some of your feedback will be addressed separately in #1590